editoast/src/views/pathfinding.rs

```diff
@@ -0,0 +1,87 @@
+use actix_web::web::{Data, Json, Path};
+use actix_web::{delete, get};
+use actix_web::{dev::HttpServiceFactory, web, HttpResponse, Responder};
+use editoast_derive::EditoastError;
+use serde::Serialize;
+use thiserror::Error;
+
+use crate::error::Result;
+use crate::models::Pathfinding;
+use crate::models::{Delete, Retrieve};
+use crate::schema::LineString;
+use crate::DbPool;
+
+#[derive(Debug, Error, EditoastError)]
+#[editoast_error(base_id = "pathfinding")]
+enum PathfindingError {
+    #[error("Pathfinding {id} does not exist")]
+    #[editoast_error(status = 404)]
+    NotFound { id: i64 },
+}
+
+/// Returns `/pathfinding` routes
+pub fn routes() -> impl HttpServiceFactory {
+    web::scope("/pathfinding").service(get_pf).service(del_pf)
+}
+
+#[derive(Clone, Debug, Serialize)]
+struct Response {
+    pub id: i64,
+    pub owner: String,
+    pub created: String,
```

Comment:
We can use `NaiveDateTime` it's serializable.

Reply:
That would change the date format in the response to ISO something: https://github.com/DGEXSolutions/osrd/pull/3767/files#diff-bd24e067d47a2da6424960aa0427161f41199a105f037f0b55036fb724290834R55

Reply:
Sound good to me.